ui/public/index.html

```diff
@@ -23,6 +23,12 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Data explorer</title>
+    <!--
+      Used for embedding Data Explorer into Terra.
+      Without this, Terra header is fixed: https://i.imgur.com/tucJzcG.gifv
+      With this, DE iframe embeds naturally: https://i.imgur.com/iqgwp8q.gifv
+    -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.1.1/iframeResizer.contentWindow.min.js"></script>
```

Comment:
I think it might be worth adding this comment from the github's readme:

> The second file (iframeResizer.contentWindow.min.js) needs placing in the page(s) contained within your iFrame. This file is designed to be a guest on someone else's system, so has no dependencies and won't do anything until it's activated by a message from the containing page.

Reply:
I think it's ok, if people are curious they can google and find that